invenio_vocabularies/cli.py

```diff
@@ -18,7 +18,8 @@
 from invenio_access import any_user
 from invenio_db import db
 
-from invenio_vocabularies.contrib.subjects.subjects import subject_record_type
+from invenio_vocabularies.contrib.subjects.subjects import \
+    record_type as subject_record_type
```

Comment:
Yeah this file needs to be changed. The create subjects vocabulary below is new to me and certainly doesn't work now. I will still do the loading in rdm-records, but I wouldn't mind eventually moving it back here if it makes sense.